File: A's/cpp/110A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    string n; cin >> n;

    int l = 0;

    for (char c : n) {
        if (c == '4' || c == '7') {
            l++;
        }
    }

    cout << (l == 4 || l == 7 ? "YES" : "NO") << endl;
    
    return 0;
}

File: A's/py/110A.py
def main() -> None:
    n: str = str(input())
    l: int = 0

    for c in n:
        if c == "4" or c == "7": l += 1

    print("YES" if (l == 4 or l == 7) else "NO")

if __name__ == "__main__":
    main()